### PR TITLE
Add outputs to ExternalSlurmDbd Stack

### PR DESCRIPTION
**Pull request by NSsirena**
### Description of changes

Add output values that should be used in cluster configuration
* SlurmDbdPrivateIP
* SlurmDbdPort
* AccountingClientSG
* SSHClientSG

### Tests
* Deployed stack in my account. The outputs were added in the stacks and the values were correct.

### References
N/A

### Checklist
- Make sure you are pointing to **the right branch**.
- If you're creating a patch for a branch other than `develop` add the branch name as prefix in the PR title (e\.g\. `[release-3.6]`).
- Check all commits' messages are clear, describing what and why vs how.
- Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py [hunk 1]
```diff
@@ -103,6 +104,8 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         #     name="slurmdbd",
         # )
 
+        self._add_outputs()
```
Comment (jdeamicis) [inline cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py:106]: Nitpick: could we call it `_add_stack_outputs()`?

File: cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py [hunk 2]
```diff
@@ -416,3 +419,30 @@ def _add_instance_to_dns(self, ip_addr, name):
...
+            self,
+            "SlurmDbdPrivateIP",
+            description="Secondary Private IP Address of the slurmdbd instance",
+            value=self.slurmdbd_private_ip.value_as_string,
+        )
+        CfnOutput(
+            self,
+            "SlurmDbdPort",
+            description="Port used to connect to slurmdbd service",
+            value="6819",  # this should be paramterized
```
Comment (jdeamicis) [inline cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py]: Nitpick: fix spelling of `parametrized`.

**Review by jdeamicis — APPROVED** [review]
Left some minor comments

File: cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py (hunk 1 above)
Comment (NSsirena) [reply cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py:106]: I followed the convention used for other PCluster stacks

**Review by jdeamicis — APPROVED** [review]

**Pull request merged**